rs/ethereum/cketh/minter/src/eth_rpc_client/mod.rs

```diff
@@ -557,9 +559,16 @@ impl<T> From<ReducedResult<T>> for Result<T, MultiCallError<T>> {
     }
 }
 
-impl From<EvmMultiRpcResult<EvmBlock>> for ReducedResult<Block> {
-    fn from(value: EvmMultiRpcResult<EvmBlock>) -> Self {
-        ReducedResult::from_internal(value).map_reduce(
+trait Reduce {
```

Comment:
I really like this trait! The code looks nicer now in my opinion.   :slightly_smiling_face: